ItunesSearch/ItunesSearch/SearchResult.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  SearchResult.swift
+//  ItunesSearch
+//
+//  Created by Elizabeth Wingate on 2/11/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import Foundation
+
+struct SearchResult: Codable {
+    var title: String
+    var creator: String
+    
+    enum CodingKeys: String, CodingKey {
+     case title = "trackName"
+     case creator = "artistName"
+    }
+}
+    struct SearchResults: Codable {
```

Comment:
remember that the type `SearchResults` is an array `[SearchResult`,  keep that in mind when we decode later.